client/package.json

```diff
@@ -16,10 +17,11 @@
     "uuid": "^3.4.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "start": "node_modules/.bin/react-scripts start",
+    "build": "node_modules/.bin/react-scripts build",
+    "test": "node_modules/.bin/react-scripts test --env=jsdom --passWithNoTests",
+    "eject": "node_modules/.bin/react-scripts eject",
+    "lint": "eslint '**/*.js' --fix "
```

Comment:
I'd suggest splitting out the `lint`ing and fixing into two separate tasks – `lint` and `lint-fix`. The CI should only ever run the `lint` task (and not modify anything).

Reply:
Update made